Makefile.am

```diff
@@ -210,29 +210,85 @@ check-doc-testapi:
 clean-local:
 	@if test -e ppmclibs/Makefile; then $(MAKE) -C ppmclibs distclean; fi
 	-rm -rf *.tar.*
-	-rm -rf t/cover_db/
-
-t/cover_db/: check
-
-.PHONY: coverage-test
-coverage-test: t/cover_db/
+	-rm -rf cover_db/
+
+EMPTY =
+TEST_FILES = \
```

Comment:
Please also adjust https://github.com/os-autoinst/os-autoinst/blob/master/dist/rpm/os-autoinst.spec#L91

Reply:
Done.
I also changed the check to
`make check test VERBOSE=1`
The build will probably a bit faster as it doesn't use Devel::Cover anymore...

Reply:
ok, can you test in OBS? If you haven't done this so far, I suggest the following:

* browse to https://build.opensuse.org/package/show/devel:openQA/os-autoinst
* click on "Branch package", click on "Accept"
* in the new project find the file "_service" and change it to point to …
  * your repo instead of "os-autoinst/os-autoinst"
  * your branch instead of "master"
* get a coffee, let it build and test :)

Reply:
done, looks good: https://build.opensuse.org/package/show/home:tinita:branches:devel:openQA/os-autoinst